Tests/checkoutlinesufo_data/expected_output/ufo2-dflt-layer.ufo/fontinfo.plist

```diff
@@ -1,162 +1,162 @@
-		<integer>6</integer>
-		<integer>3</integer>
-		<integer>5</integer>
-		<integer>4</integer>
-		<integer>5</integer>
-		<integer>2</integer>
-		<integer>2</integer>
-		<integer>4</integer>
-	</array>
-	<key>openTypeOS2TypoAscender</key>
-	<integer>730</integer>
-	<key>openTypeOS2TypoDescender</key>
-	<integer>-270</integer>
-	<key>openTypeOS2TypoLineGap</key>
-	<integer>0</integer>
-	<key>openTypeOS2UnicodeRanges</key>
-	<array>
-		<integer>0</integer>
-		<integer>1</integer>
-		<integer>2</integer>
-		<integer>7</integer>
-		<integer>9</integer>
-		<integer>29</integer>
-		<integer>32</integer>
-		<integer>33</integer>
-		<integer>57</integer>
-	</array>
-	<key>openTypeOS2VendorID</key>
-	<string>ADBO</string>
-	<key>openTypeOS2WinAscent</key>
-	<integer>1036</integer>
-	<key>openTypeOS2WinDescent</key>
-	<integer>335</integer>
-	<key>postscriptBlueFuzz</key>
-	<integer>0</integer>
-	<key>postscriptBlueScale</key>
-	<real>0.0375</real>
-	<key>postscriptBlueValues</key>
-	<array>
-		<integer>-15</integer>
-		<integer>0</integer>
-		<integer>474</integer>
-		<integer>487</integer>
-		<integer>527</integer>
-		<integer>540</integer>
-		<integer>550</integer>
-		<integer>563</integer>
-		<integer>647</integer>
-		<integer>660</integer>
-		<integer>670</integer>
-		<integer>685</integer>
-		<integer>730</integer>
-		<integer>750</integer>
-	</array>
-	<key>postscriptFamilyBlues</key>
-	<array>
-		<integer>-15</integer>
-		<integer>0</integer>
-		<integer>475</integer>
-		<integer>488</integer>
-		<integer>527</integer>
-		<integer>540</integer>
-		<integer>549</integer>
-		<integer>563</integer>
-		<integer>646</integer>
-		<integer>659</integer>
-		<integer>669</integer>
-		<integer>684</integer>
-		<integer>729</integer>
-		<integer>749</integer>
-	</array>
-	<key>postscriptFamilyOtherBlues</key>
-	<array>
-		<integer>-249</integer>
-		<integer>-239</integer>
-	</array>
-	<key>postscriptFontName</key>
-	<string>SourceSerifVariable-Roman</string>
-	<key>postscriptOtherBlues</key>
-	<array>
-		<integer>-250</integer>
-		<integer>-240</integer>
-	</array>
-	<key>postscriptStemSnapH</key>
-	<array>
-		<integer>55</integer>
-		<integer>40</integer>
-	</array>
-	<key>postscriptStemSnapV</key>
-	<array>
-		<integer>80</integer>
-		<integer>90</integer>
-	</array>
-	<key>postscriptUnderlinePosition</key>
-	<integer>-75</integer>
-	<key>postscriptUnderlineThickness</key>
-	<integer>50</integer>
-	<key>styleMapFamilyName</key>
-	<string>Source Serif Variable</string>
-	<key>styleMapStyleName</key>
-	<string>regular</string>
-	<key>styleName</key>
-	<string>Roman</string>
-	<key>trademark</key>
-	<string>Source is a trademark of Adobe Systems Incorporated in the United States and/or other countries.</string>
-	<key>unitsPerEm</key>
-	<integer>1000</integer>
-	<key>versionMajor</key>
-	<integer>1</integer>
-	<key>versionMinor</key>
-	<integer>0</integer>
-	<key>xHeight</key>
-	<integer>474</integer>
-</dict>
+  <dict>
+    <key>ascender</key>
```

Comment:
@miguelsousa I'm thinking of stripping those leading two spaces and indent the top-level element (usually `<dict>`) the same indentation level as the root `<plist>` element, just like the standard plistlib does...
it seems like a waste of characters.. On the other hand, it may introduce some extra complexity and might slow down dumping a bit.

Reply:
They don't bother me, but if you're going to change it, please do it soon.

Reply:
let's leave them like that then